### DISCUSSION TITLE
File path issue when exporting HTML from a standalone app and hosting it on the web

### DISCUSSION THREAD

**Discussion by Makunugi** [cables-gl/cables_docs#898] [Answers & Questions] [answered]
When I load a GLTF file from a local disk in a standalone app, the file path is local. I couldn't change that state, so I exported it as is and hosted it on the web, but I still got a file path error. How can I solve this?
By the way, HTML export doesn't output assets either.

![GLTFUP](https://github.com/user-attachments/assets/40fea75b-0b5f-4212-81f0-f9995c8e8630)
![filePath](https://github.com/user-attachments/assets/b6d37834-34d5-44dd-ad5c-174f78fdebe8)
![export](https://github.com/user-attachments/assets/b49e75f7-1b5b-4f39-8055-3ecde4b76d3d)
![webHostingError](https://github.com/user-attachments/assets/60df8f7d-cb9e-401c-96ff-49c4739964a6)

**Comment by steam0r** [comment]
"exported it as is" means "Export - Patch" and "html export" means "Export - HTML", both in standalone, right?

can you also tell the url you deployed to? did you deploy the patch export or the html export?

**Reply by Makunugi** [reply]
Sorry for my poor explanation.
The export is HTML export. I understand that patch export saves the Cables patch as is.
I deployed the file output by HTML export.

**Reply by Makunugi** [reply]
The third image is the file output by HTML export.

**Reply by steam0r** [reply]
awesome, thank you

**Reply by steam0r** [reply]
can you upload both these exports here? i am also not sure where that rectangle in the last screenshot comes from :)

**Comment by Makunugi** [comment]
[cables_testHTML.zip](https://github.com/user-attachments/files/17661130/cables_testHTML.zip)
[cables_testPatch.zip](https://github.com/user-attachments/files/17661131/cables_testPatch.zip)
![cables](https://github.com/user-attachments/assets/a8c13b2f-a8f5-4c9d-b17a-e522974ac673)

These are the two exported files. They should have looked like the screenshots. It seems that a box appears if the GLTF path is not set.

**Reply by Makunugi** [reply, marked as answer]
The issue was fixed by updating the standalone app. Thank you.
